src/main/resources/openapi/specs/api.kuflow.com/v1/openapi.yaml

```diff
@@ -243,7 +243,8 @@ paths:
       summary: Create a new Task in the selected Process
       description: |
         Create a Task and optionally fill its elements. We can fill in any type of element except documents.
-        If you want to add document type elements, then use the appropriate API method.
+        If you want to add document type elements, you can pass an element document indicating their 'uri' or use the appropriate API
```

Comment:
```suggestion
        If you want to add document type elements, you can pass a reference to an existing document type element indicating its 'uri'. This will copy that document into the element. In case you want to add a new document, please use the corresponding API method.
```